build_android/res/values/strings.xml

```diff
@@ -42,8 +46,14 @@
     
     <string name="preferences_ui">Storage Options</string>
     
-    <string name="pref_ui_reset_folder_title">Reset Folder</string>
-    <string name="pref_ui_reset_folder_summary">Clear stored default folder</string>
+    <string name="pref_ui_rescan_all_title">Rescan Storage</string>
+    <string name="pref_ui_rescan_all_summary">Scan storage for new games</string>
+
+    <string name="pref_ui_clear_index_title">Clear Game List</string>
+    <string name="pref_ui_clear_index_summary">Reset sll of the game\'s list and its associated details</string>
```

Comment:
Small mistake here, should be "all" instead of "sll". Should also be "game list" instead of "game's list".